lib/ansible/modules/cloud/openstack/os_server_metadata.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright (c) 2016, Mario Santos <mario.rf.santos@gmail.com>
+#
+# This module is free software: you can redistribute it and/or modify
```

Comment:
Please use the[ short GPL license](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright).

Reply:
👍 Fixed